src/pal/src/CMakeLists.txt

```diff
@@ -281,31 +281,73 @@ if(CMAKE_SYSTEM_NAME STREQUAL Linux)
     find_library(UNWIND_ARCH NAMES unwind-arm)
   endif()
 
+  if(PAL_CMAKE_PLATFORM_ARCH_ARM64)
+    find_library(UNWIND_ARCH NAMES unwind-aarch64)
+  endif()
+
   if(PAL_CMAKE_PLATFORM_ARCH_AMD64)
     find_library(UNWIND_ARCH NAMES unwind-x86_64)
   endif()
 
-  if(CLR_CMAKE_PLATFORM_ALPINE_LINUX)
+  if(CLR_CMAKE_PLATFORM_ALPINE_LINUX OR CLR_CMAKE_PLATFORM_ANDROID)
     find_library(INTL intl)
   endif()
 
-  find_library(UNWIND NAMES unwind)
+  if(NOT CLR_CMAKE_PLATFORM_ANDROID)
+    find_library(UNWIND NAMES unwind)
+
+    if(UNWIND STREQUAL UNWIND-NOTFOUND)
+      message(FATAL_ERROR "Cannot find libunwind. Try installing libunwind8-dev and libunwind8.")
+    endif(UNWIND STREQUAL UNWIND-NOTFOUND)
+
+    target_link_libraries(coreclrpal ${UNWIND})
```

Comment:
This is the reason for the test failure. The Unwind libraries must be specified after the gcc_s - that's what I have mentioned to you some time ago.